src/frontend/scripts/main.js

```diff
@@ -33,8 +33,8 @@ async function getPostsPage(perPage = 10) {
         return post;
```

Comment:
For this file, you still have changes in git.  You should pull back the version before you started working instead:

```
git checkout master src/frontend/scripts/main.js
```